Bluetooth Fast Pair: Battery Notification extension Kconfig (#17063)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * Added:

>     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_BN` Kconfig option allowing the user to enable the support for the Battery Notification extension.
>       Now, it is necessary to enable this new Kconfig option before accessing Fast Pair API elements that are associated with the Battery Notification extension.

Reviewer: ```suggestion
    * The :kconfig:option:`CONFIG_BT_FAST_PAIR_BN` Kconfig option that enables support for the Battery Notification extension.
      You must enable this option to access Fast Pair API elements associated with the Battery Notification extension.
```

Author: thanks, fixed
